File: pset1/mario/less/mario.c
// Prints mario's half pyramids

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Takes the user input
    int n;
    do
    {
        n = get_int("Height : ");
    }
    while (n > 23 || n < 0); // Reprompts the user if correct value not given

    // Prints the half pyramids
    for (int i = 1; i <= n; i++)
    {
        int cupto = n - i; // Stores the number of spaces in that row
        for (int j = 1; j <= cupto; j++)
        {
            printf(" ");
        }
        for (int k = cupto + 1; k <= (n + 1); k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/sentimental/mario/less/mario.py
# Prints mario's half pyramids
from cs50 import get_int


def main():
    # Takes the user input
    while True:
        n = get_int("Height : ")
        if n <= 23 and n >= 0:  # Repromts the user if not correct value not given
            break
    # Prints the half pyramids
    for i in range(1, n + 1):
        cupto = n - i  # Stores the number of spaces in that row
        for j in range(1, cupto + 1):
            print(" ", end="")
        for k in range(cupto + 1, n + 2):
            print("#", end="")
        print()


if __name__ == "__main__":
    main()